assets/xml/objects/object_fall.xml

```diff
@@ -1,31 +1,31 @@
             <Vtx/>
         </Array>
-        <DList Name="object_fall_DL_0011D0" Offset="0x11D0" />
-        <Texture Name="object_fall_Tex_001688" OutName="tex_001688" Format="i8" Width="64" Height="64" Offset="0x1688" />
-        <Texture Name="object_fall_Tex_002688" OutName="tex_002688" Format="i4" Width="64" Height="64" Offset="0x2688" />
-        <Texture Name="object_fall_Tex_002E88" OutName="tex_002E88" Format="i4" Width="64" Height="64" Offset="0x2E88" />
-        <DList Name="object_fall_DL_003C30" Offset="0x3C30" />
-        <Texture Name="object_fall_Tex_003E28" OutName="tex_003E28" Format="i8" Width="32" Height="64" Offset="0x3E28" />
-        <Texture Name="object_fall_Tex_004628" OutName="tex_004628" Format="i8" Width="32" Height="64" Offset="0x4628" />
-        <TextureAnimation Name="object_fall_Matanimheader_004E38" Offset="0x4E38" />
-        <TextureAnimation Name="object_fall_Matanimheader_004E58" Offset="0x4E58" />
-        <DList Name="object_fall_DL_0077F0" Offset="0x77F0" />
-        <Texture Name="object_fall_TLUT_0082F8" OutName="tlut_0082F8" Format="rgba16" Width="4" Height="4" Offset="0x82F8" />
-        <Texture Name="object_fall_TLUT_008318" OutName="tlut_008318" Format="rgba16" Width="4" Height="4" Offset="0x8318" />
-        <Texture Name="object_fall_Tex_008338" OutName="tex_008338" Format="rgba16" Width="32" Height="32" Offset="0x8338" />
-        <Texture Name="object_fall_Tex_008B38" OutName="tex_008B38" Format="rgba16" Width="64" Height="32" Offset="0x8B38" />
-        <Texture Name="object_fall_Tex_009B38" OutName="tex_009B38" Format="ci4" Width="64" Height="64" Offset="0x9B38" />
-        <Texture Name="object_fall_Tex_00A338" OutName="tex_00A338" Format="ci4" Width="64" Height="64" Offset="0xA338" />
+        <DList Name="gMoonFireballDL" Offset="0x11D0" />
+        <Texture Name="gMoonFireballFlecksTex" OutName="moon_fireball_flecks" Format="i8" Width="64" Height="64" Offset="0x1688" />
+        <Texture Name="gMoonFireballFlamesTex" OutName="moon_fireball_flames" Format="i4" Width="64" Height="64" Offset="0x2688" />
+        <Texture Name="gMoonFireballMaskTex" OutName="moon_fireball_mask" Format="i4" Width="64" Height="64" Offset="0x2E88" />
+        <DList Name="gMoonFireRingDL" Offset="0x3C30" />
+        <Texture Name="gMoonFireRingFlamesTex" OutName="moon_fire_ring_flames" Format="i8" Width="32" Height="64" Offset="0x3E28" />
+        <Texture Name="gMoonFireRingMaskTex" OutName="moon_fire_ring_mask" Format="i8" Width="32" Height="64" Offset="0x4628" />
+        <TextureAnimation Name="gMoonFireRingTexAnim" Offset="0x4E38" />
+        <TextureAnimation Name="gMoonUnusedFireballTexAnim" Offset="0x4E58" /> <!-- Almost certainly scrapped for just doing the TwoTexScrolls manually in EnFall_Fireball_Draw -->
+        <DList Name="gMoonDL" Offset="0x77F0" />
+        <Texture Name="gMoonFarSideTLUT" OutName="moon_far_side_palette" Format="rgba16" Width="4" Height="4" Offset="0x82F8" />
```

Comment:
I believe we use tlut for the OutName too.

Reply:
This is where we have to make a decision, because we are split exactly 50/50 in the codebase right now. Bigslime uses _tlut, and gmo uses _palette. All my PRs use _palette too. I got mine from OoT, which upon closer inspection is tilted in favor of "_tlut". 83 uses of "_tlut" vs. 21 uses of "_pal".

Reply:
As we discussed in discord, we decided to use _tlut for everything.